composer.json

```diff
@@ -8,11 +8,17 @@
         {
             "type": "vcs",
             "url": "https://github.com/xwp/PHP-CSS-Parser"
+        },
+        {
+            "type": "vcs",
+            "url": "git@github.com:dennisinteractive/amp-library.git"
```

Comment:
Using our fork of amp-library, see pr https://github.com/dennisinteractive/amp-library/pull/6